Vagrantfile

```diff
@@ -53,7 +41,6 @@ local_config = {
   "replication_servers" => (ENV['REPLICATION_SERVERS'] || 'false').downcase == 'true',
   "container_auto_shard" => (ENV['CONTAINER_AUTO_SHARD'] || 'true').downcase == 'true',
   "object_sync_method" => (ENV['OBJECT_SYNC_METHOD'] || 'rsync'),
-  "use_python3" => (ENV['USE_PYTHON3'] || 'true').downcase == 'true',
```

Comment:
I wonder if we might want to add an option to specify which python to bring things up under...

Eh, future work.

Reply:
Great idea for future work!  That would be very handy.